homeassistant/components/sensor/wink-eggminder.py

```diff
@@ -0,0 +1,63 @@
+"""
+homeassistant.components.egg_minder.wink
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Wink Egg Minder.
+For more details about this platform, please refer to the documentation at
+at https://home-assistant.io/components/sensor.wink/
```

Comment:
I have changed the url to be sensor.wink-eggminder, however I am not sure how you would want that added to homeassistant.io. This is different from the wink.sensor since it requires manual setup under sensor, and isn't dependent upon the wink component or hub.